robot/impl/local_robot_test.go

```diff
@@ -1676,16 +1680,20 @@ func TestConfigStartsValidReconfiguresInvalid(t *testing.T) {
 	badConfig := &config.Config{
 		Components: []resource.Config{
 			{
-				Name:                "test",
-				API:                 base.API,
-				Model:               fakeModel,
+				Name:  "test",
+				API:   base.API,
+				Model: fakeModel,
+				// Added to force a component reconfigure.
+				Attributes:          rutils.AttributeMap{"version": 1},
 				ConvertedAttributes: someConfig{},
 			},
 		},
 		Services: []resource.Config{
 			{
-				Name:                "fake1",
-				API:                 datamanager.API,
+				Name: "fake1",
+				API:  datamanager.API,
+				// Added to force a service reconfigure.
+				Attributes:          rutils.AttributeMap{"version": 1},
 				ConvertedAttributes: someConfig{},
```

Comment:
Scrolling up a bit, see how the previous `fake1` service had a `builtin.Config` object.

Reply:
That is.. minute. Great catch. I have a better sense now of what you were saying earlier today about some of these tests using `Reconfigure` to test that a single resource reconfigures by changing something small like `ConvertedAttributes`. Thanks for the comments in-line, too.